test/Microsoft.AspNet.FileProviders.Physical.Tests/PhysicalFileProviderTests.cs

```diff
@@ -25,7 +25,7 @@ public void ExistingFilesReturnTrue()
             Assert.NotNull(info);
             Assert.True(info.Exists);
 
-            info = provider.GetFileInfo("/File.txt");
```

Comment:
Need a negative test making sure that passing `/file` does the appropriate thing on each OS.